app/src/main/java/ch/hikemate/app/ui/profile/EditProfileScreen.kt

```diff
@@ -99,11 +104,11 @@ fun EditProfileScreen(
         modifier =
             Modifier.testTag(Screen.EDIT_PROFILE)
                 .padding(
-                    // Add for the status bar
                     start = 16.dp,
                     end = 16.dp,
-                    top = 16.dp,
-                ),
+                )
+                .safeDrawingPadding()
+                .verticalScroll(scrollState),
```

Comment:
Question: should you use `.imePadding()` here too? I'm not sure I understand how it works.

Reply:
Yep! I added the `imeNestedScroll` here too!